commons/src/components/ContactImage/ContactImage.svelte

```diff
@@ -0,0 +1,34 @@
+<svelte:options tag="nylas-contact-image" />
+
+<script>
+  import { beforeUpdate } from "svelte/internal";
+  import { fetchContactImage } from "@commons";
+  export let contact;
+  export let contactQuery;
+  export let height = "32px";
+  export let width = "32px";
+
+  let image;
+
+  beforeUpdate(async () => {
+    if (contact?.picture_url) {
+      image = await fetchContactImage(contactQuery, contact?.id);
```

Comment:
This maybe beyond the scope of this PR, but we'll probably also want to cache the results from our queries into the store and fetch from the store if we are making the same lookup again.

Reply:
I'll make this another story for next week